.github/workflows/rust.yml

```diff
@@ -55,7 +55,17 @@ jobs:
         if: always()
         with:
           command: clippy
-          args: --workspace --all-targets -- -D warnings
+          args: --workspace --exclude fuzz --all-targets -- -D warnings
```

Comment:
Some explanation is in order here for this hack or workaround :-)

The goal is to move `fuzz` up as a first-class package in the workspace. The challenge to it is to pass the necessary cfg flag to `fuzz` so that its dependencies are built as expected, but there's just no truly excellent way to do this I've found. 

The tradeoff here is to put some hackery into the CI and risk losing chance to be invited to the secret devops society :sunglasses:, but the benefit I see is we keep the configuration local to the package. 

Another alternative is to set configuration with `RUSTFLAGS=` but it affects everything and seems less desirable. I am open to input on this, and am not tied to this specific approach so tightly :-)

Reply:
I think a nicer solution here might be to use the `workspace.default-members` configuration to exclude the fuzz crate from default cargo commands?